otterai/main.py

```diff
@@ -1,6 +1,6 @@
 import os
 from typing import List, Dict, Any, Optional, Tuple
-from github import Github, PullRequest, PullRequestComment
+from github import Github, PullRequest, PullRequestComment, Repository
```

Comment:
🔍 Consider adding a comment explaining the purpose of importing `Repository` to enhance code readability.